src/metabase/analytics/stats.clj

```diff
@@ -124,26 +124,44 @@
     (config/config-str :database-url)        :heroku ;; Putting this last as 'database-url' seems least specific
     :else                                    :unknown))
 
+(defn appearance-ui-colors-changed?
```

Comment:
I created helper functions for these two settings as they weren't simple booleans

Reply:
In the settings we have a generic "application-colors" object, but in the UI we split them between "application colors" and "chart colors".
Applications colors are only three (brand, filter and summarize) while chart colors are a matrix of more than colors. It didn't seem allright to list them all so I subtracted the ui colors from object to see if any chart color has been customized